torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -5,9 +5,9 @@
 
 namespace c10d {
 namespace {
-c10::intrusive_ptr<ProcessGroup::Work> broadcast(
-    const c10::intrusive_ptr<ProcessGroup>& process_group,
+c10::intrusive_ptr<ProcessGroup::Work> broadcast_(
     at::TensorList tensors,
+    const c10::intrusive_ptr<ProcessGroup>& process_group,
```

Comment:
Also curious about the reason for argument ordering :)

Reply:
I think it's a convention for aten ops to have tensor-like parameters as the first argument. Especially for in-place ops, it tells that's the one get the inplace update.